syntax = "proto3";

package Protocol;

message MatchRequest{
    int32 playerId = 1;
}